medusa/databases/main_db.py

```diff
@@ -260,6 +262,7 @@ def fix_unaired_episodes(self):
                                    [common.UNAIRED, cur_unaired["episode_id"]])
 
     def fix_tvrage_show_statues(self):
+        # Remove?
         status_map = {
             'returning series': 'Continuing',
             'canceled/ended': 'Ended',
```

Comment:
@p0psicles do we have something similar to the others indexers?

"tvmaze" has the 'to be determined' status. Its not 'ended' but its not 'continuing'
Keep as it is or make a list of default medusa show status so we map all?